src/daft-dsl/src/functions/utf8/base64_decode.rs

```diff
@@ -0,0 +1,41 @@
+use common_error::{DaftError, DaftResult};
+use daft_core::prelude::*;
+
+use super::super::FunctionEvaluator;
+use crate::{functions::FunctionExpr, ExprRef};
+
+pub(super) struct Base64DecodeEvaluate {}
+
+impl FunctionEvaluator for Base64DecodeEvaluate {
```

Comment:
we've been moving away from `FunctionEvaluator` and instead implementing new functions using the `daft-functions` crate. Here's a [merged PR](https://github.com/Eventual-Inc/Daft/pull/2854/files) of migrating from the old to the new.

Reply:
Thanks for the review, let me check it.

Reply:
@universalmind303 I have submit #3101 to move utf8 from daft-dsl to dsl-functions. Then I will add those two expressions to dsl-functions. Hopefully you have time to review it.